packages/api-client/src/components/ViewLayout/ViewLayout.vue

```diff
@@ -1,6 +1,25 @@
+<style scoped>
+.view-layout > :not([hidden]):not(:first-child) {
+  border-top-width: calc(calc(var(--scalar-border-width) / 2));
+  border-left-width: 0px;
+}
+.view-layout > :not([hidden]):first-child {
+  border-bottom-width: 0px;
+}
+
+@screen xl {
+  .view-layout > :not([hidden]):first-child {
+    border-right-width: 0px;
+  }
+  .view-layout > :not([hidden]):not(:first-child) {
+    border-top-width: 0px;
+    border-left-width: calc(calc(var(--scalar-border-width) / 2));
+  }
+}
+</style>
 <template>
   <div
-    class="divide-y md:divide-y-0 divide-1 flex flex-col min-h-0 flex-1 xl:overflow-hidden md:flex-row leading-3">
+    class="view-layout flex flex-col min-h-0 flex-1 xl:overflow-hidden md:flex-row leading-3">
```

Comment:
It looks like these `divide`'s weren't actually doing anything so they can probably just be removed (unless I'm missing something).

```suggestion
    class="flex flex-col min-h-0 flex-1 xl:overflow-hidden md:flex-row leading-3">
```